templates/docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+# https://github.com/kubernetes/kubernetes/blob/master/docs/getting-started-guides/docker.md
+
+# docker run \
+#     --volume=/:/rootfs:ro \
+#     --volume=/sys:/sys:ro \
+#     --volume=/dev:/dev \
+#     --volume=/var/lib/docker/:/var/lib/docker:rw \
+#     --volume=/var/lib/kubelet/:/var/lib/kubelet:rw \
+#     --volume=/var/run:/var/run:rw \
+#     --net=host \
+#     --pid=host \
+#     --privileged=true \
+#     -d \
+#     gcr.io/google_containers/hyperkube:v1.0.6 \
+#     /hyperkube kubelet --containerized --hostname-override="127.0.0.1" \
+#     --address="0.0.0.0" --api-servers=http://localhost:8080 \
+#     --config=/etc/kubernetes/manifests
+
+kubelet:
+  image: gcr.io/google_containers/hyperkube:{{version}}
+  net: host
+  pid: host
+  privileged: true
+  volumes:
+    - /:/rootfs:ro
+    - /sys:/sys:ro
+    - /dev:/dev
+    - /var/lib/docker/:/var/lib/docker:rw
+    - /var/lib/kubelet/:/var/lib/kubelet:rw
+    - /var/run:/var/run:rw
+    - {{manifest_directory}}:/etc/kubernetes/manifests:rw
+  command: |
+    /hyperkube kubelet --containerized --hostname-override="127.0.0.1"
+    --address="0.0.0.0" --api-servers=http://localhost:8080
+    --config=/etc/kubernetes/manifests
+
+# docker run --net=host -d gcr.io/google_containers/etcd:2.0.12 \
+#   /usr/local/bin/etcd --addr=127.0.0.1:4001 --bind-addr=0.0.0.0:4001 \
+#   --data-dir=/var/etcd/data
+etcd:
```

Comment:
Unless we have a config block for single host master. this should probably just 'go away' - but lets explore that once we have HA working, then make it transition from single node setup to multi-node HA. 

Sound good? if so, lets milestone that and work towards a milestone. <3